docs/src/transient-attributes/summary.md

```diff
@@ -3,3 +3,16 @@
 Transient attributes are attributes only available within the factory
 definition, and not set on the object being built. This allows for more complex
 logic inside factories.
+
+Thse are defined using within a `transient` block:
+
+```ruby
+factory :user do
+  name { "Zero Cool" }
+  dob { age&.years.ago }
```

Comment:
Fun fact: once I had to fill a form in English and I had no idea what `dob` meant. I didn't fill in and the person explained it meant "date of birth" 😆